src/composables/useEthers.ts

```diff
@@ -14,23 +14,24 @@ type TxCallback = (
 export default function useEthers() {
   const { finalizeTransaction } = useTransactions();
   const { refetchBalances } = useAccountBalances();
+  const { supportsBlocknative } = useBlocknative();
 
   async function txListener(
     tx: TransactionResponse,
     callbacks: {
       onTxConfirmed: TxCallback;
       onTxFailed: TxCallback;
     },
-    shouldRefetchBalances = false
+    shouldRefetchBalances = true
   ) {
     try {
       const receipt = await tx.wait();
       // attempt to finalize transaction so that the pending tx watcher won't check the tx again.
-      if (APP.IsGnosisIntegration && receipt != null) {
+      if (receipt != null) {
         finalizeTransaction(tx.hash, 'tx', receipt);
       }
       callbacks.onTxConfirmed(tx);
-      if (shouldRefetchBalances) {
+      if (shouldRefetchBalances && !supportsBlocknative.value) {
         refetchBalances.value();
```

Comment:
Blocknative does a good job of refetching balances... however if it's not supported we need to refetch it manually after a successful transaction.